docs/source/quantization.rst

```diff
@@ -1009,6 +1009,9 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
 you call the `torch.ao.quantization.get_default_qconfig(backend)` or `torch.ao.quantization.get_default_qat_qconfig(backend)` function to get the default ``qconfig`` for
 ``x86`` or ``qnnpack`` backend
 
+2. If ``onednn`` backend is selected, 8 bits for activation in the default qconfig mapping ``torch.ao.quantization.get_default_qconfig_mapping('onednn')`` will be used.
+and qconfig ``torch.ao.quantization.get_default_qconfig('onednn')``. The ``onednn`` quantization backend is recommended to be used on CPUs with Vector Neural Network Instruction support.
```

Comment:
"only support VNNI CPU" is much stricter than "recommended to be used on CPUs with Vector Neural Network Instruction support."

If the `onednn` quantization backend only supports processors with VNNI, then the documentation should say that. Also, the code should warn the user when the `onednn` backend is used on a CPU without VNNI or throw an exception.

Reply:
Thanks for the comment. 
I am working on to add the warning, will keep you updated.